src/shared/components/ResourceEditor/CodeEditor.tsx

```diff
@@ -92,6 +94,15 @@ const CodeEditor = forwardRef<codemirror.Editor | undefined, TCodeEditor>(
             highlightUrlOverlay(editor);
             (ref as React.MutableRefObject<codemirror.Editor>).current = editor;
           }}
+          editorWillUnmount={() => {
+            const editor = (ref as React.MutableRefObject<
+              CodeMirror.Editor
+            >).current.getWrapperElement();
+            if (editor) editor.remove();
+            if (wrapperRef.current) {
+              (wrapperRef.current as { hydrated: boolean }).hydrated = false;
+            }
+          }}
```

Comment:
I already made these changes in #1489 to avoid rendering it twice. Just double-check before merging.